components/tooltip/tooltip.js

```diff
@@ -1007,5 +1019,20 @@ class Tooltip extends RtlMixin(LitElement) {
 		this._resizeRunSinceTruncationCheck = false;
 		target.removeChild(cloneContainer);
 	}
+
+	#onTooltipMouseEnter() {
+		if (!this.showing) return;
+		this.#isHoveringTooltip = true;
+		this._updateShowing();
+	}
+
+	#onTooltipMouseLeave() {
+		this.#isHoveringTooltip = false;
+		this.#leftTooltip = true;
+		setTimeout(() => {
+			this.#leftTooltip = false;
+			this._updateShowing();
+		}, 100); // delay to allow for mouseenter to fire if hovering on target
```

Comment:
Is there a weird race condition if they leave but then come back and leave again within this time period? Might be more obvious if you increase it to like 10s while testing. If this is a problem, you could capture the timeout identifier (the thing returned from `setTimeout`) and cancel the previous one using `clearTimeout`.

Reply:
There does seem to be; doesn't happen everytime but yeah good call, I'll fix that up.